packages/example-studio/migrations/renameField.js

```diff
@@ -0,0 +1,43 @@
+import baseClient from 'part:@sanity/base/client'
+
+// Remove when we get support for sanity exec --with-user-token
+import ConfigStore from 'configstore'
+const authToken = new ConfigStore('sanity', {}, {globalConfigPath: true}).get('authToken')
+const client = baseClient.config({token: authToken})
+// ---
+
+// Fetching documents that matches the precondition for the migration
+const fetchDocuments = () =>
+  client.fetch(`*[_type == 'author' && defined(fullname)] {_id, _rev, fullname}`)
+
+const buildPatches = docs =>
+  docs.map(doc => ({
+    id: doc._id,
+    patch: {
+      set: {name: doc.fullname},
+      unset: doc.name,
```

Comment:
Unset takes an array of fields to unset, and this seems to be using a property value instead of a field name as well